packages/cna-template/template/tsconfig.json

```diff
@@ -25,7 +25,12 @@
     },
     "types": [
       "@types/node",
+<%_ if (content) { _%>
```

Comment:
indentation can be kept because `<%_` will strip all whitespace before it

Reply:
Fixed the identation